### PR TITLE
feat: profile banner for lite user

### PR DESCRIPTION
**Pull request by labibimamhidayat**
## Describe your changes
Handle profile banner to see not connected in tab wallet and network when user is web2

## Issue ticket number / link
https://blocksphere2020.atlassian.net/browse/MYR-2623

## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [ ] If it is a core feature, I have added thorough tests.
- [x] Will this be part of a product update? If yes, please write one phrase about this update.

### CODE REVIEW

File: src/components/ProfileCard/ProfileContent.tsx [hunk 1]
```diff
@@ -143,14 +147,26 @@ export const ProfileContent: React.FC<ProfileCardProps> = props => {
             <div className={classes.column}>
               <Typography component="span">{i18n.t('Profile_Card.Network')}</Typography>
               <Typography component="span" className={classes.flex}>
-                {getSelectedIcon()} {formatNetworkTitle(currentWallet?.network)}
+                {isWeb2Users ? (
+                  'Not Connected'
```
Comment (irmannmal) [inline src/components/ProfileCard/ProfileContent.tsx:151]: Need to put this string to multi languge

File: src/components/ProfileCard/ProfileContent.tsx [hunk 2]
```diff
@@ -143,14 +147,26 @@ export const ProfileContent: React.FC<ProfileCardProps> = props => {
...
+                )}
               </Typography>
             </div>
             <div className={classes.column}>
               <Typography component="span">{i18n.t('Profile_Card.Wallet')}</Typography>
               <Typography component="span" className={classes.flex}>
-                {getSelectedIcon(true)}
-                {formatWalletTitle(currentWallet?.network)}
+                {isWeb2Users ? (
+                  'Not Connected'
```
Comment (irmannmal) [inline src/components/ProfileCard/ProfileContent.tsx:163]: And this too

**Review by irmannmal — CHANGES_REQUESTED** [review]

File: src/components/ProfileCard/ProfileContent.tsx (hunk 2 above)
Comment (labibimamhidayat) [reply src/components/ProfileCard/ProfileContent.tsx]: resolved in https://github.com/myriadsocial/myriad-web/pull/1542/commits/5388d14ad4af4a00aac4e930004ea3a7ca9981e9 @irmannmal

File: src/components/ProfileCard/ProfileContent.tsx (hunk 1 above)
Comment (labibimamhidayat) [reply src/components/ProfileCard/ProfileContent.tsx]: resolved in https://github.com/myriadsocial/myriad-web/commit/5388d14ad4af4a00aac4e930004ea3a7ca9981e9 @irmannmal

File: src/locale/ru.json [hunk 3]
```diff
@@ -226,6 +226,7 @@
     "Profile": "Профиль",
     "Logout": "Выйти",
     "Network": "Сеть",
+    "NotConnected": "Not Connected",
```
Comment (irmannmal) [inline src/locale/ru.json:229]: for temporary please use google translate

**Review by irmannmal — CHANGES_REQUESTED** [review]

File: src/locale/ru.json (hunk 3 above)
Comment (labibimamhidayat) [reply src/locale/ru.json:229]: Cykaa bylaat ? @irmannmal lol wkkwkw, ok sir

Comment (labibimamhidayat) [reply src/locale/ru.json]: resolved in https://github.com/myriadsocial/myriad-web/pull/1542/commits/fae086aeac11935ba33e61be65bdeaae36793dd4 @irmannmal

**Review by irmannmal — APPROVED** [review]

**Pull request merged**